components/button.css

```diff
@@ -0,0 +1,315 @@
+.button-container{
+    display: flex;
+    gap: 2rem;
+    align-items: center;
+    flex-wrap: wrap;
+    margin: 1rem 0rem;
+}
+
+/*--------------------- Pimary Buttons ----------------------*/
+
+.btn-text-primary{
+    border: none;
+    border-radius: 0.25rem;
+    background-color: transparent;
+    color: var(--color-barcelona-red);
+    font-size: 1rem;
+    font-weight: 600;
+    padding: 0.75rem 1rem;
+    transition: all 0.3s ease-in-out;
+    text-decoration: none;
+}
+.btn-text-primary:hover{
+    background-color: var(--color-primary-complimentary);
+    cursor: pointer;
+}
+.btn-contained-primary{
+    border: none;
+    border-radius: 0.25rem;
+    background-color: var(--color-barcelona-red);
+    color: var(--color-white);
+    padding: 0.75rem 1rem;
+    transition: all 0.2s ease-in-out;
+    text-decoration: none;
+}
+.btn-contained-primary:hover{
+    box-shadow: 0 0 5px 2px rgba(179, 42, 81, 0.55);
+    cursor: pointer;
+}
+.btn-outlined-primary{
+    border: 1px solid var(--color-barcelona-red);
+    border-radius: 0.25rem;
+    background-color: var(--color-white);
+    color: var(--color-barcelona-red);
+    padding: 0.75rem 1rem;
+    transition: all 0.3s ease-in-out;
+    text-decoration: none;
+}
+.btn-outlined-primary:hover{
+    background-color: var(--color-primary-complimentary);
+    cursor: pointer;
+}
+
+/*--------------------- Secondary Buttons ----------------------*/
+
+.btn-text-secondary{
+    border: none;
+    border-radius: 0.25rem;
+    background-color: transparent;
+    color: var(--color-barcelona-blue);
+    font-size: 1rem;
+    font-weight: 600;
+    padding: 0.75rem 1rem;
+    transition: all 0.3s ease-in-out;
+    text-decoration: none;
+}
+.btn-text-secondary:hover{
+    background-color: var(--color-medium-blue);
+    cursor: pointer;
+}
+.btn-contained-secondary{
+    border: none;
+    border-radius: 0.25rem;
+    background-color: var(--color-barcelona-blue);
+    color: var(--color-white);
+    padding: 0.75rem 1rem;
+    transition: all 0.2s ease-in-out;
+    text-decoration: none;
+}
+.btn-contained-secondary:hover{
+    box-shadow: 0 0 5px 2px rgba(44, 42, 179, 0.55);
+    cursor: pointer;
+}
+.btn-outlined-secondary{
+    border: 1px solid var(--color-barcelona-blue);
+    border-radius: 0.25rem;
+    background-color: var(--color-white);
+    color: var(--color-barcelona-blue);
+    padding: 0.75rem 1rem;
+    transition: all 0.3s ease-in-out;
+    text-decoration: none;
+}
+.btn-outlined-secondary:hover{
+    background-color: var(--color-medium-blue);
+    cursor: pointer;
+}
+
```

Comment:
Use a `btn` class for common properties, like, `padding`, `text-decoration`, `cursor`, etc..You can reduce your CSS in this way.

Reply:
Yes, I'll do that thanks.